translations/harbour-berlin-vegan-de.ts

```diff
@@ -302,5 +302,17 @@
         <oldsource>Breakfast</oldsource>
         <translation>Frühstück</translation>
     </message>
+    <message id="id-icon-toolbar-call">
+        <source>CALL</source>
```

Comment:
Wouldn't it make more sense to use normal casing and use something like `toUppercase()` in the source code? That way the casing is easier to change from the source code...